temporal-sdk/src/main/java/io/temporal/worker/MetricsType.java

```diff
@@ -135,6 +135,8 @@ private MetricsType() {}
   // gauge
   public static final String WORKER_TASK_SLOTS_AVAILABLE =
       TEMPORAL_METRICS_PREFIX + "worker_task_slots_available";
+  public static final String WORKER_TASK_SLOTS_USED =
```

Comment:
Nit: these are public, which is weird but it is what it is, so should be marked experimental.